packages/neuron-ui/package.json

```diff
@@ -42,21 +42,24 @@
   "dependencies": {
     "@nervosnetwork/ckb-sdk-core": "0.103.1",
     "@nervosnetwork/ckb-sdk-utils": "0.103.1",
+    "ahooks": "^3.7.5",
```

Comment:
Unnecessary external deps should be avoided cuz they may be hacked

Ref: https://blog.npmjs.org/post/180565383195/details-about-the-event-stream-incident

Besides, deps should be pinned to avoid automatic upgrades

Reply:
got it, i will remove it.